services/app/src/components/team/team.css

```diff
@@ -1,16 +1,20 @@
 .team-container {
   box-sizing: border-box;
   border: 1px solid var(--dark-border);
-  flex: none;
   order: 1;
   width: 384px;
+  height: 843px;
```

Comment:
How did you came up with this value?

Reply:
Team container's height should be controlled by `GameLayoutSideArea` component

Reply:
> How did you came up with this value?

I came up with this value, by adding chessboard height, chess pieces and other components. Since our chessboard and pieces heights are constant, I figured it makes sense